src/main/resources/db/migration/V63__add_event_logging_table.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+----------------------------------------
+
+DROP TABLE IF EXISTS event_logging;
+CREATE TABLE IF NOT EXISTS event_logging
+(
+    id                                      UUID        PRIMARY KEY,
+    before_data                             JSONB       DEFAULT NULL,
+    processed_data                          JSONB       DEFAULT NULL,
+    source_system_id                        TEXT        DEFAULT NULL,
+    uuid                                    UUID        DEFAULT NULL,
+    data_source                             TEXT        DEFAULT NULL,
```

Comment:
Not sure what this is but it will probably become clearer when the code which uses it goes in!
You can safely ignore these comments until then I reckon